examples/wind/src/layers/wind-layer/wind-layer.js

```diff
@@ -166,24 +165,21 @@ export default class WindLayer extends Layer {
   }
 
   createTexture(gl, opt) {
```

Comment:
same as above, let opt = {} by default.
but how do we control the texture size without passing down the width and height?